src/main.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+var todoItems;
+
+var Module = (function () {
+
+    var _getHtmlItem = function (item) {
+        var htmlItem = `<input type="checkbox" class="task_list__item" name="task" value="${item.id}" ${ item.completed ? 'checked' : ''}> ${item.title} <br>`;
+        return htmlItem;
+    };
+
+    var fetchAll = async function () {
+
```

Comment:
El Async aqui no es necesario ya que no estas usando esta funcion como una funcion asincrona (usando awaits) sino que la usas de manera sincrona/lineal al final de este archivo. Echale una ojeada a este [articulo](https://derickbailey.com/2017/06/06/3-features-of-es7-and-beyond-that-you-should-be-using-now/) especificamente la seccion de "Async Functions".